tests/lib/rules/no-fallthrough.js

```diff
@@ -129,6 +129,14 @@ ruleTester.run("no-fallthrough", rule, {
             errors: errorsDefault
         },
         {
+            code: "switch(foo) { case 0: // comment\n default: b() }",
+            errors: errorsDefault
+        },
+        {
+            code: "switch(foo) { case 0: /* comment */ default: b() }",
+            errors: errorsDefault
+        },
```

Comment:
I'd feel better if there were test cases for what was initially reported (`case 0:\n    // comment\ncase 1: yay();`, that is, with a line break between the case and the comment). Would you mind terribly adding those cases?